guides/common/modules/proc_upgrading-rhel7-to-rhel8.adoc

```diff
@@ -0,0 +1,28 @@
+[id="Upgrading-Project-from-RHEL7-to-RHEL8_{context}"]
+= Upgrading {Project} From {RHEL} 7 to {RHEL} 8
+
+Use this procedure to upgrade your {Project} installation from {RHEL} 7 to {RHEL} 8.
+
+.Prerequisites
+* {Project} {ProjectVersion} is supported on both {RHEL} 7 and {RHEL} 8.
+* Subsequent minor versions of {Project} 6 will potentially be supported on {RHEL} 8 only.
+* If you have {Project} {ProjectVersion} running on {RHEL} 7, you must migrate to {RHEL} 8 to stay up-to-date on {Project} {ProjectVersion} minor releases.
+* {Project} {ProjectVersion} running on {RHEL} 7.
+* An offline backup created by `satellite-maintain`.
+This can be an older full backup with incremental backups leading to the point from where you want to restore.
+	*** Not supported by `satellite-clone`.
```

Comment:
This isn't a standard way of writing anything. I don't know what happened here. Please restructure and reword. What is this? A title? A sentence? Which sentence does it act on, the previous or the following? Very very unclear.